README.md

```diff
@@ -1,13 +1,15 @@
-<h1>Light and Leadership Non-profit Project</h1>
+Light and Leadership Non-profit Project
```

Comment:
@RachellCalhoun So this page is a `.md` which stands for markdown

Markdown is more or less a simplified version of html with its own rules.

Check it out at https://guides.github.com/features/mastering-markdown/